ts-web-component/src/setup-replicache.ts

```diff
@@ -24,10 +25,12 @@ export async function setupReplicache(): Promise<Replicache<M>> {
     window.location.href = "/listId/" + listId;
   }
 
+  // See https://doc.replicache.dev/licensing for how to get a license key.
+  const licenseKey = import.meta.env.VITE_REPLICACHE_LICENSE_KEY;
+  assert(licenseKey, "Missing VITE_REPLICACHE_LICENSE_KEY");
+
   const r = new Replicache<M>({
-    // See https://doc.replicache.dev/licensing for how to get a license key.
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
eslint was not configured so this didn't do anything.

Changed it to a runtime assert instead